planutils/packages/ff/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Fast-Forward",
+    "description": "https://fai.cs.uni-saarland.de/hoffmann/ff.html",
+    "install-size": "1.1M",
+    "dependencies": [],
+    "endpoint": {
+        "services": {
+            "solve": {
```

Comment:
Should probably just use the template, like lama-first:
https://github.com/AI-Planning/planutils/blob/main/planutils/packages/lama-first/manifest.json

This is what it inserts: https://github.com/AI-Planning/planutils/blob/main/planutils/packages/TEMPLATE/SERVICE_TEMPLATE/planner.json

Reply:
Oh, I get it.